partcad/src/partcad/sentry.py

```
@@ -0,0 +1,135 @@
+
+                for key, value in kwargs.items():
+                    if isinstance(value, (str, int, float, bool)):
+                        span.set_attribute(key, value)
+                return await func(*args, **kwargs)
+
+        return wrapper
+
+    return decorator
+
+
+def instrument(exclude: list = []):
```

Ruff (0.8.2)

Missing return type annotation for public function `instrument`

(ANN201)